examples/llm/mcloud/mcli_pretokenize_oci_upload.yaml

```diff
@@ -0,0 +1,26 @@
+run_name: c4-2k-pre-tokenized
+gpu_type: a100_40gb
+gpu_num: 8
+cluster: r0z0 # Update with your cluster here!
+image: mosaicml/composer:0.12.1
+
+integrations:
+- integration_type: pip_packages
+  # Install necessary oci-cli
+  packages:
+  - oci-cli
```

Comment:
had no clue this was a thing! Lets add an `oci-cli` version or atleast a `<??` thing though.